Hospital.iml

```diff
@@ -31,8 +21,11 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
+<<<<<<< HEAD
```

Comment:
asta nu inseamna conflict?

Reply:
sterge-l si pune unu' de la un commit mai vechi. .iml era bun sa-l punem in .gitignore